conode/go.mod

```diff
@@ -12,4 +12,6 @@ require (
 	google.golang.org/appengine v1.6.5 // indirect
 )
 
+replace go.dedis.ch/cothority/v3 => ../cothority
```

Comment:
Debug?

Reply:
No debug this time, I want it to use the cothority checked out locally. Most of the time it's not the same as master...